ehsm_kms_service/constant.js

```diff
@@ -40,27 +40,33 @@ const ehsm_keyorigin_t = {
     EH_EXTERNAL_KEY: 2
 }
 
-const ehsm_paddingMode_t = {
-    EH_NO_PADDING: 0,
-    EH_PAD_RSA_PKCS1: 1,
-    EH_PAD_RSA_NO: 3,
-    EH_PAD_RSA_PKCS1_OAEP: 4,
-    EH_PAD_RSA_PKCS1_PSS: 6

```

Comment:
padding mode is deleted? 
will encrypt/decrypt still use padding mode?

Reply:
mistake delete, fix